lib/CMakeLists.txt

```diff
@@ -502,18 +529,9 @@ if(QUDA_BACKWARDS)
     SOURCE comm_common.cpp
     APPEND
     PROPERTY COMPILE_DEFINITIONS ${BACKWARD_DEFINITIONS})
-  set_property(
-    SOURCE comm_common.cpp
-    APPEND
-    PROPERTY COMPILE_DEFINITIONS QUDA_BACKWARDSCPP)
-  set_property(
-    SOURCE malloc.cpp
-    APPEND
-    PROPERTY COMPILE_DEFINITIONS ${BACKWARD_DEFINITIONS})
-  set_property(
-    SOURCE malloc.cpp
-    APPEND
-    PROPERTY COMPILE_DEFINITIONS QUDA_BACKWARDSCPP)
+  set_property(SOURCE comm_common.cpp APPEND PROPERTY COMPILE_DEFINITIONS QUDA_BACKWARDSCPP)
```

Comment:
Note to @hummingtree , you'll probably need to reconcile this in #1020, though it should be easy b/c it appears this change only converted multi-line commands to single line.

Reply:
This is not from me :)

Reply:
That was cmake-format ;-)